kubernetes-response-engine/deployment/falco-config/falco.yaml

```diff
@@ -0,0 +1,102 @@
+# File(s) or Directories containing Falco rules, loaded at startup.
```

Comment:
Instead of copying the entire falco config, can you just include a patch or highlight the changes that need to be made from the main falco config to add the additional file output? I'm worried that with a second copy it will become out-of-date with respect to the main config file as we make changes to the main config file.

Reply:
As long as the new Falco Helm Chart is merged (I hope in a while), we can remove the falco and falco_config directory from deployment. So this change will dissapear.

Thanks!